client/components/AvailabilityGrid/AvailabilityGrid.js

```diff
@@ -198,12 +198,11 @@ class AvailabilityGrid extends React.Component {
 
       participants.forEach((participant) => {
         if (participant.availability.indexOf(cellFormatted) > -1) {
-          availableOnDate.push(participant.name);
+          availableOnDate.push(participant.userId.name);
```

Comment:
Why is everything under the "userId" property: that should just be the user's id, not the full user object.